.travis.yml

```diff
@@ -14,8 +14,8 @@ matrix:
       before_script: createdb htest
       script: tox
       after_success:
-        tox -e coverage
-        tox -e codecov
+        tox -e py27-coverage
+        tox -e py27-codecov
 
```

Comment:
Commands like `tox -e coverage` etc (without any `pyXY`) will work, but I think they'll just use the default Python on the user's system. For consistency across systems we always want to list them explicitly.